onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -444,20 +444,21 @@ def aten_arctanh(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-@torch_op("aten::argmax", trace_only=True)
-def aten_argmax(self: TReal, dim: Optional[int] = None, keepdim: bool = False) -> TReal:
+@torch_op("aten::argmax")
+def aten_argmax(self: TReal, keepdim: bool = False) -> TInt:
```

Comment:
```suggestion
def aten_argmax(self: TTensor, keepdim: bool = False) -> INT64:
```

good catch; to refine the types further: